components/eamxx/tests/single-process/ace/input.yaml

```diff
@@ -0,0 +1,70 @@
+%YAML 1.1
+---
+driver_options:
+  atmosphere_dag_verbosity_level: 5
+
+time_stepping:
+  time_step: ${ATM_TIME_STEP}
+  run_t0: ${RUN_T0}
+  number_of_steps: ${NUM_STEPS}
+
+atmosphere_processes:
+  atm_procs_list: [ace]
+  ace:
+    checkpoint_path: /pscratch/sd/m/mahf708/emulator/ace_ckpt_traced${GPU_OR_CPU}.tar
+    sc2ace_mapfile: /global/homes/m/mahf708/misc/map_ne30pg2_to_gaussian_180x360_latlon.nc
+    ace2sc_mapfile: /global/homes/m/mahf708/misc/map_gaussian_180x360_latlon_to_ne30pg2.nc
+grids_manager:
+  Type: ACE
+  grids_names: [AceLL,AcePG,Physics]
+  ACE:
+    aliases: [AceLL,AcePG,Physics]
+    type: point_grid
+    num_longitude_points: 360
+    num_latitude_points: 180
+    num_vertical_levels: 8
+    gaussian_grid: true
+    data_filename: /global/homes/m/mahf708/xgnc/sims/random/eamxx_latlon_grid.nc
+    sc_ncolumns: 21600
```

Comment:
It seems redundant to pass ncolumns here. We can prob infer it from the $sc_data_filename file...

Reply:
Also sc_nlevels seems redundant with `num_vertical_levels` above.